src/lib/operators/join_hash.cpp

```diff
@@ -375,17 +376,17 @@ void probe(const RadixContainer<RightType>& radix_container,
 
   for (size_t current_partition_id = 0; current_partition_id < (radix_container.partition_offsets.size() - 1);
        ++current_partition_id) {
+    const auto& partition_begin = radix_container.partition_offsets[current_partition_id];
+    const auto& partition_end = radix_container.partition_offsets[current_partition_id + 1];
+
+    // Skip empty partitions to avoid empty output chunks
+    if ((partition_end - partition_begin) == 0) {
```

Comment:
isn't this just a complicated way of writing: `partition_end == partition_begin`?